src/mower_logic/cfg/MowerLogic.cfg

```diff
@@ -15,6 +15,7 @@ gen.add("tool_width", double_t, 0, "Width of the mower", 0.14, 0.1, 2)
 gen.add("enable_mower", bool_t, 0, "True to enable mow motor", False)
 gen.add("manual_pause_mowing", bool_t, 0, "True to disable mowing automatically", False)
 gen.add("current_area", int_t, 0, "Current Mow Area", 0, 0, 99)
+gen.add("active_semiautomatic_task", bool_t, 0, "True if currently mowing the map, False when not running", False)
```

Comment:
why did you choose to have it as a parameter in parameter server instead of a global variable in mower_logic?

Reply:
i followed the logic of current_area. Will move it